### PR TITLE
[New3D] Render CameraInfo topics

### PR DESCRIPTION
**Pull request by jhurliman**
**User-Facing Changes**

- None

**Description**

- CameraInfo renderable
- Don't attempt to compute bounding spheres for point clouds
- Improve frame selection when moving between different data sources
- Remove stereo camera ban

Screen Shot 2022-05-12 at 6 36 32 PM

### CODE REVIEW

**Comment by defunctzombie** [comment]
Such excite

File: packages/studio-base/src/panels/ThreeDeeRender/ThreeDeeRender.tsx [hunk 1]
```diff
@@ -256,12 +259,26 @@ export function ThreeDeeRender({ context }: { context: PanelExtensionContext }):
...
         }
       }
+
+      // Choose the root frame with the most children
+      const rootsToCounts = new Map<string, number>();
+      for (const frame of curRenderer.transformTree.frames().values()) {
+        const rootId = frame.root().id;
+        rootsToCounts.set(rootId, (rootsToCounts.get(rootId) ?? 0) + 1);
+      }
+      const rootsArray = Array.from(rootsToCounts.entries());
```
Comment (foxymiles) [inline packages/studio-base/src/panels/ThreeDeeRender/ThreeDeeRender.tsx:277]: Looks like you could use lodash `countBy` here?

**Review by foxymiles — APPROVED** [review]

File: packages/studio-base/src/panels/ImageView/lib/PinholeCameraModel.ts [hunk 2]
```diff
@@ -54,42 +90,75 @@ export default class PinholeCameraModel {
...
+    const fy = P[5];
+    const cx = P[2];
+    const cy = P[6];
+
+    let x = (point.x - cx) / fx;
+    let y = (point.y - cy) / fy;
+
+    const x0 = x;
+    const y0 = y;
+    for (let i = 0; i < iterations; i++) {
```
Comment (jtbandes) [inline packages/studio-base/src/panels/ImageView/lib/PinholeCameraModel.ts:134]: what is this algorithm? a comment would be appreciated for curious readers.

Comment (jhurliman) [reply packages/studio-base/src/panels/ImageView/lib/PinholeCameraModel.ts:134]: https://github.com/foxglove/studio/pull/3425

**Pull request merged**